src/waddle/audios/align_offset_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+import tempfile
+import wave
+
+import numpy as np
+
+from waddle.audios.align_offset import (
+    align_speaker_to_reference,
+    find_offset_via_cross_correlation,
+    shift_audio,
+)
+
+DEFAULT_SR = 48000
+
+
+def generate_test_audio(sr=DEFAULT_SR, s_offset=0, e_offset=0):
+    """Generate a test sine wave audio signal with optional start and end offset."""
```

Comment:
not a big deal, but didn't understand what `s_offset` and `e_offset` do. Maybe "padding" or "spacing" is the right word? Or you can just write "generate a test audio with `s_offset` seconds of silence, followed by 1 second of sine wave, followed by `e_offset` seconds of silence."

Reply:
Thank you for your review.
You are right, so I changed it to use padding and revised the comment to be more detailed.